commons/passive/garmin/garmin_generic_calories.avsc

```diff
@@ -0,0 +1,30 @@
+{
+   "name":"GarminGenericCalories",
+   "type":"record",
+   "namespace":"org.radarcns.passive.garmin",
+   "doc":"Gives the information about the calories consumed.",
+   "fields":[
+
+       {
+            "name":"time",
+            "type":"double",
+            "doc":"Last updated time."
+         },
+         {
+            "name":"timeReceived",
+            "type":"double",
+            "doc":"Device receiver timestamp in UTC (s)."
+         },
+      {
+         "name":"currentActiveCalories",
+         "type":"int",
+         "doc":"Current active calories."
+      },
+      {
+         "name":"currentTotalCalories",
```

Comment:
You can remove `current` here and other fields too.

Reply:
Done. Available in latest commit.